charts/consul/test/unit/gossip-encryption-autogeneration-job.bats

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bats
+
+load _helpers
+
+@test "gossipEncryptionAutogeneration/Job: disabled by default" {
+  cd `chart_dir`
+  assert_empty helm template \
+      -s templates/gossip-encryption-autogen-job.yaml  \
```

Comment:
I think every test in this file needs to get updated with this change
```suggestion
      -s templates/gossip-encryption-autogeneration-job.yaml  \
```

Reply:
Great catch. I let this one slip by unchanged when I modified the filename.